### PR TITLE
chore: Add White JS Logo SVG

### PR DESCRIPTION
**Pull request by ChTiSh**
## Description

Added the white JS Logo to the site assets.
Logo named js-white.svg can be found at https://github.com/nodejs/nodejs.org/tree/main/public/static/images/logos.

## Validation

Verified on StoryBook

### Check List

- [Checked] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [Checked] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [Checked] I have run `npx turbo format` to ensure the code follows the style guide.
- [Checked] I have run `npx turbo test` to check if all tests are passing.
- [n/a ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: components/__design__/logos.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const JSLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <div>
```
Comment (ovflowd) [inline components/__design__/logos.stories.tsx]: I wonder if we need `<div>` 2 times? Maybe only one div container is enough?

**Review by ovflowd — APPROVED** [review]
LGTM! Thank you for your first contribution! 🎉 ✨ 

I've left one tiny comment 👀

File: components/__design__/logos.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const JSLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+       <div>
```
Comment (ovflowd) [inline components/__design__/logos.stories.tsx]: I feel that the indentation is wrong here, I assume you edited this file directly on GitHub UI?

Could you pull your latest changes and run `npm run format` to ensure that the file is properly formatted?

File: components/__design__/logos.stories.tsx (hunk 1 above)
Comment (ChTiSh) [reply components/__design__/logos.stories.tsx]: Good point, just remove the unnecessary <div> here

File: components/__design__/logos.stories.tsx (hunk 2 above)
Comment (ChTiSh) [reply components/__design__/logos.stories.tsx]: Will do, and yes, I did edit it directly on Github. Sorry about making this so complicated.

Comment (ovflowd) [reply components/__design__/logos.stories.tsx]: No worries! Completely understandable 😄

**Pull request merged**